src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -189,6 +198,8 @@ def what(self) -> What:
         return What.UNKNOWN
 
     def sql_migrate_external(self, target_table_key):
+        if self.what == What.MANAGED_EXTERNAL:
```

Comment:
create a separate method for this and call it in table migrator, otherwise it's a bug